zipline/finance/commission.py

```diff
@@ -69,6 +69,18 @@ def calculate(self, order, transaction):
         """
         raise NotImplementedError('calculate')
 
+    def __eq__(self, other):
+        return self.asdict() == other.asdict()
+
+    def __hash__(self):
+        return hash((
+            type(self),
+            tuple(sorted(iteritems(self.asdict())))
```

Comment:
Barring some known reason to keep them, my tendency would be to remove 'em. Probably run all integration tests to check though. @ssanderson Any opinion?